test/utils/node-test.js

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
Please remove this

Reply:
done [08f162d](https://github.com/nodejs/undici/pull/2570/commits/08f162dfc8ea131892cea782e115edcd5a13adb7)